### ISSUE TITLE
Collision box disappearance

### ISSUE THREAD

**Issue by Collinvh** [relavis/VehiclesPlus#43]
**Describe the bug**  
When driving randomly the hitbox can dissapear and you can't drive up anything but can phase thru it to right about the seat 

**To Reproduce**  
1. Drive (yeah sorry i've got no clue why it's doing this)

**Expected behavior**  
The hitbox to be calculated correctly

**Screenshots or Video Recordings**  
Don't have any

**Server and Plugin Information**  
 - Installed plugins: ArmorStandTools, Essentials, EssentialsChat, EssentialsSpawn, LuckPerms, Multiverse-Core, ProtocolLib, (My own custom plugin), Vault, VehiclesPlusPro, WorldEdit
 - VehiclesPlus Version: 2.0.9
 - Pro or Lite? Pro
 - Server Platform and Version [eg: **PaperSpigot** 1.12.2 #1618]: git-Paper-1618 (MC: 1.12.2) 
 - Operating System (if applicable): Linux (Ubuntu)

**Additional context**  
[16:45:57] [Server thread/WARN]: [VehiclesPlusPro] Task #520 for VehiclesPlusPro v2.0.9 generated an exception
java.util.NoSuchElementException: null
	at org.bukkit.craftbukkit.v1_12_R1.util.UnsafeList$Itr.next(UnsafeList.java:248) ~[patched_1.12.2.jar:git-Paper-1618]
	at net.minecraft.server.v1_12_R1.Chunk.a(Chunk.java:1015) ~[patched_1.12.2.jar:git-Paper-1618]
	at net.minecraft.server.v1_12_R1.World.getEntities(World.java:2630) ~[patched_1.12.2.jar:git-Paper-1618]
	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.getNearbyEntities(CraftWorld.java:782) ~[patched_1.12.2.jar:git-Paper-1618]
	at me.legofreak107.vehiclesplus.vehicles.managers.CollisionManager.handleEntityCollision(CollisionManager.java:216) ~[?:?]
	at me.legofreak107.vehiclesplus.vehicles.managers.CollisionManager.checkCollision(CollisionManager.java:101) ~[?:?]
	at me.legofreak107.vehiclesplus.vehicles.vehicles.objects.SpawnedVehicle.update(SpawnedVehicle.java:210) ~[?:?]
	at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:64) ~[patched_1.12.2.jar:git-Paper-1618]
	at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:423) ~[patched_1.12.2.jar:git-Paper-1618]
	at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:840) ~[patched_1.12.2.jar:git-Paper-1618]
	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:423) ~[patched_1.12.2.jar:git-Paper-1618]
	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:774) ~[patched_1.12.2.jar:git-Paper-1618]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:666) ~[patched_1.12.2.jar:git-Paper-1618]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_262]

**Comment by relavis** [comment]
Agh. It's always 1.12 that gives me issues lol. I'll take a look into this.

**Comment by relavis** [comment]
Should be fixed in 2.1.0

**Comment by stijnb1234** [comment]
I'm on 2.1.0-dev3, but I still got this issue.

```
[14:14:29 WARN]: [VehiclesPlusPro] Task #124 for VehiclesPlusPro v2.1.0-dev3 generated an exception
java.util.NoSuchElementException: null
        at org.bukkit.craftbukkit.v1_12_R1.util.UnsafeList$Itr.next(UnsafeList.java:248) ~[patched_1.12.2.jar:git-Paper-1618]
        at net.minecraft.server.v1_12_R1.Chunk.a(Chunk.java:1015) ~[patched_1.12.2.jar:git-Paper-1618]
        at net.minecraft.server.v1_12_R1.World.getEntities(World.java:2630) ~[patched_1.12.2.jar:git-Paper-1618]
        at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.getNearbyEntities(CraftWorld.java:782) ~[patched_1.12.2.jar:git-Paper-1618]
        at me.legofreak107.vehiclesplus.vehicles.managers.CollisionManager.handleVehicleCollision(CollisionManager.java:156) ~[?:?]
        at me.legofreak107.vehiclesplus.vehicles.managers.CollisionManager.checkCollision(CollisionManager.java:105) ~[?:?]
        at me.legofreak107.vehiclesplus.vehicles.vehicles.objects.SpawnedVehicle.update(SpawnedVehicle.java:213) ~[?:?]
        at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:64) ~[patched_1.12.2.jar:git-Paper-1618]
        at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:423) ~[patched_1.12.2.jar:git-Paper-1618]
        at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:840) ~[patched_1.12.2.jar:git-Paper-1618]
        at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:423) ~[patched_1.12.2.jar:git-Paper-1618]
        at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:774) ~[patched_1.12.2.jar:git-Paper-1618]
        at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:666) ~[patched_1.12.2.jar:git-Paper-1618]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_275]
````

**Comment by stijnb1234** [comment]
Implemented the same fix for this error. Testing it now.

**Comment by stijnb1234** [comment]
Fixed in 2.1.0.
